recipe/meta.yaml

```
@@ -24,97 +22,93 @@ source:
     folder: src/gpl
 
 {% set lc, vs, ve, prefix, parbuild = "\\", "$", "", "$PREFIX", "-j $CPU_COUNT" %}  # [unix]
-{% set lc, vs, ve, prefix, parbuild = "^", "%", "%", "%LIBRARY_PREFIX%", "" %}      # [win]
 build:
-  number: 1002
-  features:
-    - blas_{{ blas_variant }}  # [not win]
-  skip: true  # [win32 or (win and vc<14)]
+  number: 0
+  skip: true  # [win32 or win]
   script: |
     mkdir build
     cd build
-
-    cmake ..                                 {{ lc }}
-      -G"{{vs}}CMAKE_GENERATOR{{ve}}"        {{ lc }}
-      -DCMAKE_PREFIX_PATH="{{ prefix }}"     {{ lc }}
-      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"  {{ lc }}
-      -DCMAKE_BUILD_TYPE=Release             {{ lc }}
-      -DBUILD_EXAMPLES=OFF                   {{ lc }}
-      -DBUILD_META_EXAMPLES=OFF              {{ lc }}
-      -DBUNDLE_JSON=OFF                      {{ lc }}
-      -DBUNDLE_NLOPT=OFF                     {{ lc }}
-      -DUSE_SVMLIGHT=OFF                     {{ lc }}
-      -DENABLE_TESTING=OFF                   {{ lc }}
-      -DENABLE_COVERAGE=OFF                  {{ lc }}
-      -DUSE_SWIG_DIRECTORS=ON
-    if errorlevel 1 exit 1  # [win]
-
+    cmake ..                                                  {{ lc }}
+      -G"{{vs}}CMAKE_GENERATOR{{ve}}"                         {{ lc }}
+      -DCMAKE_PREFIX_PATH="{{ prefix }}"                      {{ lc }}
+      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"                   {{ lc }}
+      -DCMAKE_BUILD_TYPE=Release                              {{ lc }}
+      -DBUILD_EXAMPLES=OFF                                    {{ lc }}
+      -DBUILD_META_EXAMPLES=OFF                               {{ lc }}
+      -DBUNDLE_JSON=OFF                                       {{ lc }}
+      -DBUNDLE_NLOPT=OFF                                      {{ lc }}
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}
+      -DENABLE_TESTING=OFF                                    {{ lc }}
+      -DENABLE_COVERAGE=OFF                                   {{ lc }}
+      -DUSE_SWIG_DIRECTORS=ON                                 {{ lc }}
+      -DLICENSE_GPL_SHOGUN=OFF                                {{ lc }}  # [bla_vendor == 'Intel']
+      -DBLA_VENDOR={{ bla_vendor }}
     cmake --build . --target shogun --config Release -- {{ parbuild }}
-    if errorlevel 1 exit 1  # [win]
-
     cmake --build . --target install --config Release -- {{ parbuild }}
-    if errorlevel 1 exit 1  # [win]
-
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - python
-    - cmake
+    - cmake =3.15
   host:
-    - arpack       # [not win]
+    - arpack
     - bzip2
     - ctags
-    - curl
+    - libcurl
+    - libgomp       # [linux]
     - eigen
     - glpk
     - hdf5
-    - json-c       # [not win]
+    - json-c 0.11
```

Oops. We only have 0.12.
```suggestion
    - json-c 0.12
```

---

oh nooo :) should i still add the range or simply just go with the fixed 0.12?
btw in case i just commit the suggestion (didn't no about this new feature) we dont need the re-render?

---

Just go with the fixed 0.12

> we dont need the re-render?

Yes, no need to rerender for this.